assignment.js

```diff
@@ -57,17 +64,27 @@ Use the following test cases to confirm your program meets the success criteria
 2. Given the following:
     problem1Numbers = [9, 18, 62, 3, 17]
 
-   Result: 0
+   Result: 21.8
 
 ************/
 
 console.log('Problem 2:')
 
 // Add your code below this line
 
+let numbers = [1, 2, 3] // INSERT ANY NUMBERS
+let sum = 0
+let average
+
+numbers.forEach(function(number) {
```

Comment:
@carly-ga great job! `.reduce()` could also be used to quickly calculate the sum as well

`numbers.reduce((sum, number) => sum + number)`